src/librustc/cfg/graphviz.rs

```diff
@@ -22,11 +22,11 @@ pub struct LabelledCFG<'a, 'tcx: 'a> {
 impl<'a, 'tcx> LabelledCFG<'a, 'tcx> {
     fn local_id_to_string(&self, local_id: hir::ItemLocalId) -> String {
         assert!(self.cfg.owner_def_id.is_local());
-        let node_id = self.tcx.hir().hir_to_node_id(hir::HirId {
+        let hir_id = hir::HirId {
             owner: self.tcx.hir().def_index_to_hir_id(self.cfg.owner_def_id.index).owner,
             local_id
-        });
-        let s = self.tcx.hir().node_to_string(node_id);
+        };
+        let s = self.tcx.hir().hir_to_string(hir_id);
```

Comment:
If the original `node_to_string` is still around, please rename this to `node_to_string_from_hir_id` or similar.
The `node_` means "HIR node" not `NodeId`.

Reply:
`node_to_string` is removed with this PR. Shall I rename `hir_to_string` to `node_to_string` now?